### ISSUE TITLE
Colored tabs or Tab division by connection.

### ISSUE THREAD

**Issue by moonjoungyoung** [dbeaver/dbeaver#10235]
**Is your feature request related to a problem? Please describe.**
It is too hard to indistinguishable many tabs.

**Describe the solution you'd like**
Colored tabs / Tab division by connection.

I saw this requests at past, but it's been a long time since then.
Is there a solution now?

https://dbeaver.io/forum/viewtopic.php?f=3&t=916
https://github.com/dbeaver/dbeaver/issues/300
https://github.com/dbeaver/dbeaver/issues/1777

**Comment by uslss** [comment]
thanks for the suggestion

**Comment by ppython** [comment]
This feature would be awesome.

To illustrate what I would need (And what @moonjoungyoung described) : 
![dbeaver_colors](https://user-images.githubusercontent.com/18702503/104487215-d44be400-559a-11eb-9502-ad7961e50801.png)
- The areas in 1, 2 and 3 are colored
 - The tabs in 4 are not, and as the screenshot shows: To which connection each tab belong to? we can't know without opening them.

This feature would solve this problem 😃

**Comment by LoloIrie** [comment]
Agree, any option (colors or tabs grouped by connexion) would be **really** helpful.

**Comment by ppython** [comment]
bumping this one :)

**Comment by xavt** [comment]
This would be greatly benefic; for example, use cases where one has several similar/identic databases and not able to distinguish betweeb tabs.

**Comment by bmoAtolcd** [comment]
I need this feature "group tab by connection", it will be very useful. 
Often I too open several databases with same name on different servers.

**Comment by LoloIrie** [comment]
Really nice would be a solution like the Chrome Browser to group tabs. Thanks

**Comment by Sarke** [comment]
@bmoAtolcd and @LoloIrie , something like this: #3027 ?

**Comment by svonjoi** [comment]
There is any temporal workaround to somehow difference 2 open tables from different sources, without switching to each one?

**Comment by bmoAtolcd** [comment]
> @bmoAtolcd and @LoloIrie , something like this: #3027 ?

interesting but not realy that i want.
I need something that group tab by connection, perhaps a tab by connection with have child-tabs.

**Comment by xavt** [comment]
Newer issue duplicating this one:
https://github.com/dbeaver/dbeaver/issues/16827

**Comment by xavt** [comment]
related issue:
- https://github.com/dbeaver/dbeaver/issues/1352

**Comment by iraidaa** [comment]
We have the same request from our customer. Internal support ticket number is 464515.
